src/components/context-menu-component.ts

```diff
@@ -0,0 +1,39 @@
+import * as html from '@hyperapp/html'
+import {getLogger} from './../logger'
+
+const logger = getLogger('main')
+
+export const state = {
+  menuOpen: false,
+}
+
+export const actions = {
+  toggleMenu: ({menuOpen: !state.menuOpen}),
+}
+
+export function view(
+  _state: any,
+  _actions: any,
+  items: Map<string, () => void> = new Map([

```

Comment:
extract into a constant (if map will never be modified/mutated at runtime) or a function to return a new map each time if it will be mutated possible at some point